CHANGES.md

```diff
@@ -50,3 +50,7 @@ made private.
 
 - Simplified `util/shlib_wrap`; if your shell doesn't do "$@" properly
 when there are no arguments, please look at an earlier version.
+
+- Cherry-picked changes to assembly generation to support running code
```

Comment:
is it worth saying the original source of the cherry-pick?

Reply:
Yes, it's probably better to elaborate on the provenance. I'll rephrase this when I pull the corresponding arm64 changes.